static/app/bootstrap/initializeSdk.tsx

```diff
@@ -185,7 +185,9 @@ export function initializeSdk(config: Config) {
 
       // attach feature flags to the event context
       if (event.contexts) {
-        const flags = FeatureObserver.singleton().getFeatureFlags();
+        const flags = FeatureObserver.singleton({
+          bufferSize: FEATURE_FLAG_BUFFER_SIZE,
+        }).getFeatureFlags();
```

Comment:
could skip passing in the `bufferSize` value here, the default value will be fine to rely on. it already optional
```suggestion
        const flags = FeatureObserver.singleton({}).getFeatureFlags();
```